pyglmnet/pyglmnet.py

```diff
@@ -13,6 +13,8 @@
 from .externals.sklearn.utils import check_random_state, check_array, check_X_y
 from .externals.sklearn.utils.validation import check_is_fitted
 
+from tqdm import tqdm
```

Comment:
I would question if we really need to add this dependency. Granted it's a light one but still it's better to avoid when possible.

Reply:
I agree with this point. I've seen that pyglmnet has not many dependencies and since tqdm is a light one I thought that it wouldn't hurt too much. However, since this is basically a really small feature, I guess we could make all of this optional for the final user (e.g., did you install tqdm? good, then you can have better progress visualization. If you don't have tqdm, then we fall back to the original method.)

Reply:
yes agree with approach, you can have a nested import then. It might slow down import of pyglmnet then. But probably not by much